graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -15,24 +16,55 @@
 MergedSchemaDescriptor = namedtuple(
     'MergedSchemaDescriptor', (
         'schema_ast',  # Document, AST representing the merged schema
-        'name_to_schema_id',  # Dict[str, str], type name to id of the schema the type is from
+        'type_name_to_schema_id',  # Dict[str, str], mapping type name to the id of its schema
     )
 )
 
 
-def merge_schemas(schema_id_to_ast):
-    """Check that input schemas do not contain conflicting definitions, then merge.
+CrossSchemaEdgeDescriptor = namedtuple(
+    'CrossSchemaEdgeDescriptor', (
+        'edge_name',  # str, name used for the corresponding in and out fields
+        'outbound_field_reference',  # FieldReference namedtuple for the outbound field
```

Comment:
How about source_field_reference instead?

Reply:
I think this goes back to the source/sink vs out/in issue again, here I believe we want out/in rather than source/sink since we're describing the edge as a whole, not from the perspective of one end.